lib/ssl/doc/src/using_ssl.xml

```diff
@@ -41,46 +41,51 @@
   'tlsv1.3')</c> </seemfa>.  The available cipher suites for a
   connection depend on the TLS version and pre TLS-1.3 also on the
   certificate. To see the default cipher suite list change <c>all</c>
-  to <c>default</c>.  Note that TLS 1.3 and previous versions does not
+  to <c>default</c>.  Note that TLS 1.3 and previous versions do not
   have any cipher suites in common, for listing cipher suites for a
   specific version use <seemfa
   marker="ssl:ssl#cipher_suites/2"><c>ssl:cipher_suites(exclusive,
   'tlsv1.3')</c> </seemfa>.  Specific cipher suites that you want your
   connection to use can also be specified. Default is to use the
   strongest available.</p>
-  
+
+
+  <p>The following sections shows small examples of how to set up
+  client/server connections using the Erlang shell. The returned
+  value of the <c>sslsocket</c> is abbreviated with <c>[...]</c> as
+  it can be fairly large and is opaque to the user except for the
+  purpose of pattern matching.</p>
+
+
+  <note><p>Note that client certificate verification is optional for the server and needs additional conguration
+    on both sides to work. Also note that the option certs_keys is introduced in OTP-25.
+  </p>
+  </note>
+
   <section>
-    <title>Setting up Connections</title>
-    
-    <p>This section shows a small example of how to set up
-    client/server connections using the Erlang shell. The returned
-    value of the <c>sslsocket</c> is abbreviated with <c>[...]</c> as
-    it can be fairly large and is opaque.</p>
-    
-    <section>
-      <title>Minimal Example</title>
-      
-      <note><p> The minimal setup is not the most secure setup of TLS/DTLS.</p>    
-      </note>
+    <title>Basic Client</title>
+   <code type="erl"> ssl:start(), ssl:connect("google.com", 443, [{cacerts, public_key:cacerts_get()}, {verify, verify_peer}]).
+   ok</code>
+  </section>
 
-      <p>To set up client/server connections:</p>
+  <section>
+    <title>Basic Connection</title>
 
-      <p><em>Step 1:</em> Start the server side:</p>
-      <code type="erl">1 server> ssl:start().
+    <p><em>Step 1:</em> Start the server side:</p>
+    <code type="erl">1 server> ssl:start().
 ok</code>
-      
-      <p><em>Step 2:</em> Create a TLS listen socket: (To run DTLS add the option {protocol, dtls})</p>
-      <code type="erl">2 server> {ok, ListenSocket} =
-ssl:listen(9999, [{certfile, "cert.pem"},
-                  {keyfile, "key.pem"},
-                  {reuseaddr, true}]).
-{ok,{sslsocket, [...]}}</code>
 
-
-      <p><em>Step 2: From OTP-25 it is equivalent to</em> </p>
+     <p><em>Step 2:</em> with alternative certificates,
+    in this example the EDDSA certificate will be preferred if TLS-1.3
+    is negotiated and the RSA certificate will always be used for TLS-1.2
+    as it does not support the EDDSA algorithm: Added in OTP-25 </p>
```

Comment:
```suggestion
    as it does not support the EDDSA algorithm.</p>
```
I don't think the "Added in OTP-25" is adding any value here

Reply:
Agree. I included the change. No via Github as I have already done other changes to my branch!